scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -1678,6 +1918,9 @@ def get_user_agent_fragment(self):
             ver = k8s_cache.get_api_server_version()
             if ver:
                 ver = 'k8s=%s' % ver
+            runtime = k8s_cache.get_container_runtime()
```

Comment:
I packaged the most recent code into a k8s image and deployed to minikube.
I see this error on startup and the the k8s & k8se log files remain 0 length.

![image](https://user-images.githubusercontent.com/48775713/56391710-518acf00-61e4-11e9-8e9b-935947a59d2b.png)

I think at least one thing to fix is to set `self._container_runtime = None` in `KubernetesCache`

Reply:
Yes.  That was an oversight.  I've also added `agent_container_id = None` too, both of which should be set to `None` in `__init__`.